### ISSUE TITLE
[proposed Label theming] Unable to add theme apps into a solution to be used across environments

### ISSUE THREAD

**Issue by charrisonfuji** [microsoft/coe-starter-kit#951]
### Describe the issue

Hi, when trying to add the app template as below into a solution I get the following error. The question I want to raise is that can a new app created from the app templates be included in another solution to be used across environments ? Or do you have to import the master theme solution into each environment and just work from the app in there and it cannot be moved to other environments ? 

Please can I have answer to this in the next couple of days as this is important to know for projects etc. 

![image](https://user-images.githubusercontent.com/84843401/128007609-66d840ff-529d-43c1-814c-134290bf3959.png)

### Expected Behavior

To be able to add the msapp file into a solution to then be pushed to other environments 

### What solution are you experiencing the issue with?

Theming

### What solution version are you using?

1.2

### What app or flow are you having the issue with?

Table app template 

### Steps To Reproduce

save as (apptemplatetablet) created new app
Then create a new solution and then try to add the new app into that solution
![image](https://user-images.githubusercontent.com/84843401/128008073-277d5df1-f11c-4b2f-8fea-3f584a81efba.png)

**Comment by charrisonfuji** [comment]
@SPB21
@devkeydet

**Comment by charrisonfuji** [comment]
Hi @slaouist , @mikefactorial could really do with any update if at all possible please? Even to just say how long you estimate it will take to get back to me. 

 All I need is clarification on whether it's possible for apps using theming to be included in solutions so that they can be moved to other environments. As I would like to use this on commerical projects across environments.  
 
  Thanks

**Comment by mikefactorial** [comment]
@charrisonfuji based on the error the app in question has already been added to another solution. Instead of adding the app from the Outside solutions tab. Look for it in the From solutions tab. Tried to reproduce the behavior you are seeing in this screen capture.
![Untitled Project](https://user-images.githubusercontent.com/42348035/128343096-95afaf76-07be-48c0-ba62-cbd4dd0ee0ed.gif)

**Comment by charrisonfuji** [comment]
Hi @mikefactorial thanks for the response but thats what I am already doing. Attached video to show you what I am doing. 
![GithubDemo mkv](https://user-images.githubusercontent.com/84843401/128486336-95d07eb3-fd52-44bb-b771-14c7a5a1d8ed.gif)

**Comment by mikefactorial** [comment]
@charrisonfuji I think you missed the context of the screen recording I sent. The error you are getting is because you're adding it from the Outside Dataverse tab. Switch to From Dataverse tab and look for the app there to add to your solution

**Comment by devkeydet** [comment]
@mikefactorial - He can't because it's not showing up in the "From Deataverse" tab.
![image](https://user-images.githubusercontent.com/4499296/128511632-afc18b4d-8cc8-4ab6-9c45-67068817fbfe.png)
It's only showing up in the "Outside Dataverse" tab.

@charrisonfuji - I was able to repro you situation.  I was also able to work around it going to the Solution the app is already in and clicking "Publish all customizations".  Once I did that, I was able to go to another Solution and see the app in the "From Dataverse" tab when trying to add it.  I will point the teams who own this experience in make.powerapps.com to this thread for comment.  I agree this is a confusing experience.

by publishing customizations This is not related to the Theming solution in the CoE Starter Kit, this is
